finality-aleph/src/sync/service.rs

```diff
@@ -395,14 +397,21 @@ where
                         target: LOG_TARGET,
                         "Error marking block as imported: {}.", e
                     )
+                } else {
+                    // TODO: use is_best_block info from Forest
+                    if number > self.metrics.get_best_block() {
+                        self.metrics.update_best_block(number);
```

Comment:
push this logic into metrics code

Reply:
agreed & done, hope the todo will be resolved quite soon anyway